Microsoft/microsoft-intune/_meta/smart-descriptions.json

```
@@ -1,3 +1,66 @@
-{
-
-}
\ No newline at end of file
+[  {
+    "value": "Operation {action.name} made on device {host.name}",
+    "conditions": [
+      {
+        "field": "host.name"
+      },
+      {
+        "field": "action.name"
+      }
+    ],
+    "relationships": [
+      {
+        "source": "host.name",
+        "target": "action.name",
+        "type": "made"
+      }
+    ]
+  },
+  {
+    "value": "Operation {action.name} made by {user.name}",
+    "conditions": [
+      {
+        "field": "user.name"
+      },
+      {
+        "field": "action.name"
+      }
+    ],
+    "relationships": [
+      {
+        "source": "user.name",
+        "target": "action.name",
+        "type": "made"
+      }
+    ]
+  },
+    {
+    "value": "{user.name} on device {host.name}",
+    "conditions": [
+      {
+        "field": "host.name"
+      },
+      {
+        "field": "user.name"
+      }
+    ],
+    "relationships": [
+      {
+        "source": "user.name",
+        "target": "host.name",
+        "type": "logged on"
+      }
+    ]
+  },
+  {
+    "value": "{host.name} compliant status is {intune.compliant}",
```

```suggestion
    "value": "{host.name} compliance status evaluated at {intune.compliant}",
```

I would have say compliance :thinking: